browse-scaler/src/__test__/resize.test.js

```diff
@@ -1,55 +1,71 @@
-const fs = require('fs');
-const sharp = require('sharp');
-const { resizeImage, notFound } = require('../resize');
+const fs = require("fs");
```

Comment:
Why did these change to double quotes?

Reply:
the linter config files didn't copy over for some reason - that's been fixed